apps/marketplace/pages/DownloadReports/DownloadReports.scss

```diff
@@ -0,0 +1,9 @@
+.inputsContainer {
+    display: flex;
+    margin-top: 0.75rem;
+  }
+  
+.inputsContainer div {
```

Comment:
Should we nest the `div` styles in the above block?

Reply:
file deleted